### PR TITLE
Add Toggl

### PR DESCRIPTION
**Pull request by arturs105**
## Add Toggl

- [x] I have read the [contributing guidelines](https://github.com/poteto/hiring-without-whiteboards/blob/master/CONTRIBUTING.md)
- [x] I agree to the [Code of Conduct](https://github.com/poteto/hiring-without-whiteboards/blob/master/CODE_OF_CONDUCT.md)
- [x] I have followed the [format](https://github.com/poteto/hiring-without-whiteboards/blob/master/CONTRIBUTING.md#format) prescribed in the contributing guidelines
- [x] (OPTIONAL) In your pull request message, add additional context on the interview process if necessary

### CODE REVIEW

**Comment by langri-sha** [comment]
Reviewed guidelines, ack content speaks the truth, checked the MD output :mag:, confirmed there are no references to t-shirts.

Should we squish commits if everything is ok? LGTM :rocket:!

File: README.md [hunk 1]
```diff
@@ -581,6 +581,7 @@ Additions to this document that are properly formatted will automatically be pus
 - [Tilde](http://www.tilde.io) | Portland, OR | Pair programming sessions with each member of the team, working on problems similar to daily work.
 - [Timbuktu](https://www.timbuktutravel.com) | Cape Town, South Africa | On site interview and pair programming exercise
 - [Titanium](https://titanium.codes) | Moldova, Chisinau | High level review of public activity on GitHub/BitBucket/Gitlab (if applicable) and screening via phone, On-site technical & Team fit interview, Formal "Meet the Team" meeting
+- [Toggl](https://toggl.com/jobs) | Remote / Tallinn, Estonia | Online test on basic programming skills, followed by interview (typically includes get-to-know questions and technical skill testing). Depending on the team, there may be a take-home or live coding assignment.  **Paid test week** to work with the team on actual bug/features.
```
Comment (heytherewill) [inline README.md]:
```suggestion
- [Toggl](https://toggl.com/jobs) | Remote / Tallinn, Estonia | Online test on basic programming skills, followed by interview (typically includes get-to-know questions and technical skill testing). Depending on the team, there may be a take-home or live coding assignment.  **Paid test week** to work with the team on actual bugs/features.
```

Comment (poteto) [reply README.md]: @arturs105 do you want to fix this typo?

Comment (arturs105) [reply README.md]: Typo fixed, this is ready for merging now.

**Comment by poteto** [comment]
bors retry

**Pull request merged**